test/plp/SearchResultsProvider.test.js

```diff
@@ -174,7 +174,7 @@ describe('SearchResultsProvider', () => {
         await wrapper.update()
       })
 
-      expect(fetch).toHaveBeenCalledWith('/api/test?__v__=development')
+      expect(fetch).toHaveBeenCalledWith('/api/test')
```

Comment:
Why was this test changed?

Reply:
It broke when adding the sameOrigin test to addVersion, but I can revert the test now that I've removed the sameOrigin test from addVersion.